### PR TITLE
Removing IVT where possible

### PR DESCRIPTION
**Pull request by Zombach**
https://github.com/dotnet/aspire/issues/3949

Made some minor edits and also cleaned up IVT if possible.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3978)

### CODE REVIEW

**Comment by Zombach** [comment]
@eerhardt Have a look, please. And let me know if this makes sense.

File: src/Aspire.Hosting.AWS/AWSProvisioningException.cs [hunk 1]
```diff
@@ -8,6 +8,5 @@ namespace Aspire.Hosting.AWS;
 /// </summary>
 /// <param name="message"></param>
 /// <param name="innerException"></param>
-public class AWSProvisioningException(string message, Exception? innerException = null) : Exception(message, innerException)
-{
-}
+public class AWSProvisioningException(string message, Exception? innerException = null)
```
Comment (eerhardt) [inline src/Aspire.Hosting.AWS/AWSProvisioningException.cs:11]: Can you revert any unnecessary whitespace changes?

File: tests/cosmos.module.bicep [hunk 2]
```diff
@@ -0,0 +1,42 @@
+targetScope = 'resourceGroup'
```
Comment (eerhardt) [inline tests/cosmos.module.bicep:1]: What is this file for / from? I think it can be deleted.

File: src/Aspire.Hosting.AWS/AWSProvisioningException.cs (hunk 1 above)
Comment (Zombach) [reply src/Aspire.Hosting.AWS/AWSProvisioningException.cs:11]: Yes OK. I thought whitespace would be useful.

Comment (eerhardt) [reply src/Aspire.Hosting.AWS/AWSProvisioningException.cs:11]: No, in general unnecessary changes shouldn't be made in PRs. A single PR should stick to a single set of changes. Making unnecessary changes not only makes the reviewer do more work, but it also clutter source control history.

Comment (Zombach) [reply src/Aspire.Hosting.AWS/AWSProvisioningException.cs:11]: I understood you. Sorry for the mistake. I'll know for the future.

**Comment by Zombach** [comment]
I'll do it again. To keep it clean
